Inc/HALAL/Services/ADC/ADC.hpp

```diff
@@ -7,25 +7,45 @@
 
 #pragma once
 #include "ST-LIB.hpp"
+#include "C++Utilities/CppUtils.hpp"
 
 #define ADC_BUF1_LEN 4
 #define ADC_BUF2_LEN 2
 #define ADC_BUF3_LEN 2
 
-struct ADCchannel {
-	ADC_HandleTypeDef* adc;
-	uint8_t rank;
+#define LPTIM1_PERIOD 3000
+#define LPTIM2_PERIOD 3000
+#define LPTIM3_PERIOD 3000
+
+struct buffer { // Will be moved into appropiate file when include structure is fixed.
```

Comment:
Add TODO: in the comment so that the cube reminds you and doesn't miss it.

Reply:
done